docs/wcf-config.md

```diff
@@ -0,0 +1,168 @@
+# WCF Instrumentation Configuration
+
+## WCF Client Configuration (.NET Framework)
+
+Add the `IClientMessageInspector` instrumentation via a behavior extension on
+the clients you want to instrument:
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Q.: just to understand where this configuration requirement comes from? Is it possible to change the OTel WCF package to not require this config?

Reply:
This comes from contrib WCF implementation, so not here anyway